<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pixel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 8bit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12bit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 12bit</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_StartValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>557</y>
      <width>650</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1024</number>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_WindowWidth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>650</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Start">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Star:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Width">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load3D">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 3D</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_LayerNr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>650</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>129</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Layer">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>650</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
